contracts/secret-storage/SecretManager.sol

```diff
@@ -184,6 +184,10 @@ contract SecretManager is
 
     uint256 public secretId;
 
+    mapping(address => uint256[]) public storeSecretIds;
```

Comment:
it should be property of secret store itself

Reply:
updated